<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlottingManager</class>
 <widget class="QWidget" name="PlottingManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plotting Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QSplitter" name="splitterLeft">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QScrollArea" name="scrollTop">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollTopContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>720</width>
          <height>263</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="leftTopLayout">
         <item>
          <widget class="QLabel" name="lblPlotMode">
           <property name="text">
            <string>Plot Mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbPlotMode">
           <item>
            <property name="text">
             <string>2D Plot</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>3D Plot</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSettings">
           <property name="text">
            <string>Settings</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="treeSettings">
           <property name="headerHidden">
            <bool>true</bool>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QScrollArea" name="scrollBottom">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollBottomContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>720</width>
          <height>263</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="leftBottomLayout">
         <item>
          <widget class="QLabel" name="lbl3DGroups">
           <property name="text">
            <string>3D Groups</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="tree3DGroups">
           <property name="maximumHeight">
            <number>150</number>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>En.</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Del.</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lblSignals">
           <property name="text">
            <string>Available Signals</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listSignals">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
     <widget class="QWidget" name="rightPane">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="PlotCanvas" name="plotCanvas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotCanvas</class>
   <extends>QWidget</extends>
   <header>plot_canvas.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
